jason-lab/lib/error-middleware.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function (err, req, res, next) {
```

Comment:
`eslint` doesn't like that this `next` parameter is unused; however, if you remove it, Express can't tell the difference between `function (err, req, res)` and `function (req, res, next)`. So we have to keep the unused parameter:

```javascript
// eslint-disable-next-line no-unused-vars
module.exports = function (err, req, res, next) {
```